src/Navbar.test.js

```diff
@@ -23,8 +22,35 @@ afterEach(() => {
 });
 
 describe("Navbar", () => {
-  it("renders a navbar with text", () => {
-    expect.assertions(1);
+  const mockSessionToken = {
+    userId: 1,
+    token: "3ecb9d1d-863f-4207-b076-d868e6544c3b"
+  };
+
+  it("renders a logged-in navbar with text", () => {
+    expect.assertions(5);
+
+    const history = createMemoryHistory();
+    history.push("/");
+
+    let wrapper;
+    act(() => {
+      wrapper = mount(
+        <Router history={history}>
+          <Navbar sessionToken={mockSessionToken} />
+        </Router>
+      );
+    });
+
+    expect(wrapper.debug()).toMatchSnapshot();
```

Comment:
why are you checking the snapshot of the `debug()`? 
Why not just `expect(wrapper).toMatchSnapshot();`

Reply:
We're using `debug()` here because it seems to obscure the `history` parameters which change for every time we run the tests and so the snapshot becomes useless when we don't use `debug()`.